.archon/scripts/quality-check.py

```
@@ -0,0 +1,136 @@
+
+from _common import log, probe, read_json, work_dir, write_json
+
+EXPECT_W, EXPECT_H = 1080, 1920
+DURATION_TOLERANCE = 1.0   # seconds of drift from the narration we accept
+MIN_MEAN_DB = -32.0        # quieter than this and the voice is inaudible
+MAX_PEAK_DB = -0.5         # louder than this and we are clipping
+N_FRAMES = 6
+
+
+def ffmpeg_stderr(args: list[str]) -> str:
+    return subprocess.run(
```

Ruff (0.16.1)

[error] `subprocess` call: check for execution of untrusted input

(S603)